.github/workflows/pull_request_lint.yml

```diff
@@ -19,3 +19,9 @@ jobs:
           titleRegex: '^#(?<ticketNumber>\d+)'
           bodyRegex: '#(?<ticketNumber>\d+)'
           bodyURLRegex: 'http(s?):\/\/(github.com)(\/LambdaTest)(\/test-at-scale)(\/issues)\/(?<ticketNumber>\d+)'
+
+      - name: PR Label Check
+        uses: docker://agilepathway/pull-request-label-checker:latest
```

Comment:
does this run in a docker container? How much build time/credits will it consume? Is it justified to use that much amount of credits for such a simple check?